### PR TITLE
Enable mambabuild

### PR DESCRIPTION
**Pull request by markotoplak**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by markotoplak** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -72,6 +72,7 @@ requirements:
     - xlrd >=1.2.0
     - xlsxwriter
     - {{ pin_compatible('numpy') }}
+    - filelock >=3.8.0  # remove when new canvas-core is released
```
Comment (PrimozGodec) [inline recipe/meta.yaml:75]:
```suggestion
```
The new version of orange-canvas-core is already on conda-forge so it can be removed.

Comment (markotoplak) [reply recipe/meta.yaml:75]: Thanks. But then I'd have to change the canvas-core requirement and I would not want to do it in the same release.

Comment (PrimozGodec) [reply recipe/meta.yaml:75]: Makes sense.

**Pull request merged**